scripts/bump-version.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+sed -i "s/^version =.*/version = $2/" setup.cfg
```

Comment:
need to double check this will do what you want -- since I don't think this package is using setup.cfg